### DISCUSSION TITLE
PiJuice Zero on Radxa Zero

### DISCUSSION THREAD

**Discussion by ollax452** [PiSupply/PiJuice#914] [Q&A] [answered]
Hi all!
Is it possible to run the PiJuice Zero on a Radxa Zero? The i2c pins are the same and I don't need the EEPROM.
I have installed the Debian Buster and started with manual installation: sudo dpkg -i ./pijuice-base_1.7.all.deb 

But the dtparam is not available for this unit. 
Any ideas how to install the software package?

Best Regards
Olle

**Comment by tvoverbeek** [comment, marked as answer]
Pijuice Zero only needs I2C and the power pins, so in principle you should be able to get it to work on Radxa Zero.
Look at the post install script (https://github.com/PiSupply/PiJuice/blob/master/Software/Source/debian-base/postinst)
how I2C is handled. You have to use the corresponding Radxa Zero commands to set I2C.

**Reply by ollax452** [reply]
Thanks!
